resources/schema.py

```diff
@@ -643,6 +646,11 @@ def resolve_piers(self, info, **kwargs):
             "harbor__municipality__translations",
         ).select_related("harbor", "harbor__availability_level", "harbor__municipality")
 
+    def resolve_harbor_by_servicemap_id(self, info, **kwargs):
+        return Harbor.objects.filter(
+            servicemap_id=kwargs.pop("servicemap_id", None)
```

Comment:
you csan just do `kwargs.get("servicemap_id")` or `kwargs["servicemap_id"]` since it's now required and kwargs are not used for any object updates after this line 🙂